cves/CVE-2012-3439.yml

```diff
@@ -116,9 +127,11 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
+  answer: | 
+    This vulnerability was first reported to the Tomcat security team by Tilmann Kuhn.
+    This is listed on the Tomcat website.
```

Comment:
Can you give more detail about how was this vulnerability discovered? or there is no evidence or content talk about how this vulnerability discovered.

Reply:
There were no details given